entities/package.py

```diff
@@ -0,0 +1,188 @@
+        super().__init__('drill', drill)
+
+
+class FootprintPad():
+    def __init__(self, uuid: str, side: Side, shape: Shape, position: Position,
+                 rotation: Rotation, size: Size, drill: Drill):
+        self.uuid = uuid
+        self.side = side
+        self.shape = shape
+        self.position = position
+        self.rotation = rotation
+        self.size = size
+        self.drill = drill
+
+    def __str__(self) -> str:
+        ret = '(pad {} {} {}\n'.format(self.uuid, self.side, self.shape) +\
+            ' {} {} {} {}\n)'.format(self.position, self.rotation, self.size, self.drill)
+        return ret
+
+
+class Footprint():
+    def __init__(self, uuid: str, name: Name, description: Description):
+        self.uuid = uuid
+        self.name = name
+        self.description = description
+        self.pads = []  # type: List[FootprintPad]
+        self.polygons = []  # type: List[Polygon]
+        self.texts = []  # type: List[StrokeText]
+
+    def add_pad(self, pad: FootprintPad) -> None:
+        self.pads.append(pad)
+
+    def add_polygon(self, polygon: Polygon) -> None:
+        self.polygons.append(polygon)
+
+    def add_text(self, text: StrokeText) -> None:
+        self.texts.append(text)
+
+    def __str__(self) -> str:
+        ret = '(footprint {}\n'.format(self.uuid) +\
+            ' {}\n'.format(self.name) +\
+            ' {}\n'.format(self.description)
+        ret += indent_entities(self.pads)
+        ret += indent_entities(self.polygons)
+        ret += indent_entities(self.texts)
+        ret += ')'
+        return ret
+
+
+class Package:
+    def __init__(self, uuid: str, name: Name, description: Description,
+                 keywords: Keywords, author: Author, version: Version,
+                 created: Created, deprecated: Deprecated, category: Category):
+        self.uuid = uuid
+        self.name = name
+        self.description = description
+        self.keywords = keywords
+        self.author = author
+        self.version = version
+        self.created = created
+        self.deprecated = deprecated
+        self.category = category
+        self.pads = []  # type: List[PackagePad]
+        self.footprints = []  # type: List[Footprint]
+
+    def add_pad(self, pad: PackagePad) -> None:
+        self.pads.append(pad)
+
+    def add_footprint(self, footprint: Footprint) -> None:
+        self.footprints.append(footprint)
+
+    def __str__(self) -> str:
+        ret = '(librepcb_package {}\n'.format(self.uuid) +\
+            ' {}\n'.format(self.name) +\
+            ' {}\n'.format(self.description) +\
+            ' {}\n'.format(self.keywords) +\
+            ' {}\n'.format(self.author) +\
+            ' {}\n'.format(self.version) +\
+            ' {}\n'.format(self.created) +\
+            ' {}\n'.format(self.deprecated) +\
+            ' {}\n'.format(self.category)
+        ret += indent_entities(self.pads)
+        ret += indent_entities(self.footprints)
```

Comment:
Same thing here, sort pads and footprints.

Reply:
Same as above.